### DISCUSSION TITLE
`revealjs`: `.fragment ` `divs` within lists - howto avoid vertical space?

### DISCUSSION THREAD

**Discussion by r-cheologist** [quarto-dev/quarto-cli#7340] [Q&A] [answered]
### Description

Consider a slide like so:

```qmd
---
format:
  revealjs
---

# Slide Title
1. Item 1

::: {.fragment .fade-out}
2. Item 2
3. Item 3
:::
```

This will work to the extend that after `quarto render` the resulting `*.html` will a) contain an ordered list and b) items 2 and 3 will fade out on moving through the presentation, **but** the empty line prior to the `div` invocation is required for it to work and introduces more vertical white space between items 1 and 2.

How may the aditional white space be avoided?

**Comment by mcanouil** [comment, marked as answer]
You can't, or at least not this way.

The following are two correct syntaxes for two different things as described in the documentation (<https://pandoc.org/chunkedhtml-demo/8.7-lists.html>).

<table>
<tr><th>Spaced</th><th>Compact</th></tr>
<tr>
<td>

```md
1. Item 1

2. Item 2

3. Item 3
```

</td>
<td>

```md
1. Item 1
2. Item 2
3. Item 3
```

</td>
</tr>
</table>

See the answer in:
- #1459

Also a bit related:
- #3293

**Comment by r-cheologist** [comment]
Thanks for those hints! The "hacky" explicit wrapping of the whole list into `<ul>...</ul>`as proposed in #1459 works for me.
